mordant/src/commonMain/kotlin/com/github/ajalt/mordant/terminal/TerminalInfo.kt

```diff
@@ -9,11 +9,11 @@ import com.github.ajalt.mordant.rendering.AnsiLevel
  * [width] and [height] don't automatically change if the terminal is resized. Call
  * [updateTerminalSize] to get the latest values.
  */
-class TerminalInfo(
+data class TerminalInfo(
     /** The terminal width, in cells */
-    width: Int,
+    var width: Int,
```

Comment:
This is a change in API, but I don't see any harm in it, and I like using the data class for `toString`.